variables.tf

```diff
@@ -53,24 +62,14 @@ variable "azure_region" {
     condition     = length(var.azure_region) > 0
   }
 }
-variable "created_by_tag" {
-  description = "Tag to use when naming resources."
+variable "costcenter_tag" {
```

Comment:
This name is confusing. This is not the tag, it's the tag value. At least change the description.

Reply:
I'll refactor variable names. I see the same inconsistency for "created_by_tag" variable in this module.